tutorials/math/vectors_advanced.rst

```diff
@@ -214,7 +212,7 @@ point_b will work since they are in the same plane:
     // this works the same
     // var D = normal.Dot(pointB);
 
-Doing the same in 3D is a little more complex and will be explained
+Doing the same in 3D is a little more complex and is explained
 further down.
```

Comment:
Is this really explained below? 3D is not mentioned on this page below this point.

If we mean the `Collision detection in 3D` section above or another place, we should add a link here or delete this sentence.

```suggestion
in :ref:`Collision detection in 3D`.
```